source/_patterns/04-components/menu/menu--main/_menu--main.scss

```diff
@@ -43,7 +43,7 @@
     background-color: gesso-color(ui, generic, background-dark);
     display: none;
     left: 0;
-    margin-top: gesso-spacing(1);
+    margin: rem(gesso-spacing(1)) 0 0 0;
```

Comment:
[minor nitpick]

This should be `margin: rem(gesso-spacing(1)) 0 0;`. I’m surprised prettier didn’t catch it.